br/pkg/lightning/mydump/parquet_parser.go

```diff
@@ -446,6 +451,11 @@ func setDatumByString(d *types.Datum, v string, meta *parquet.SchemaElement) {
 	if meta.LogicalType != nil && meta.LogicalType.DECIMAL != nil {
 		v = binaryToDecimalStr([]byte(v), int(meta.LogicalType.DECIMAL.Scale))
 	}
+	if meta.Type != nil && *meta.Type == parquet.Type_INT96 && len([]byte(v)) == 96/8 {
+		ts := int96ToTime([]byte(v))
+		ts = ts.UTC()
+		v = ts.Format("2006-01-02 15:04:05.999999Z")
```

Comment:
this variable can be extracted as a constant

Reply:
1f379a1